aspnetcore/tutorials/signalr/includes/signalr6.md

```diff
@@ -196,52 +197,53 @@ The preceding highlighted code adds SignalR to the ASP.NET Core dependency injec
 
 ## Add SignalR client code
 
-1. Replace the content in `Pages/Index.cshtml` with the following code:
+Replace the content in `Pages/Index.cshtml` with the following code:
 
-   [!code-cshtml[Index](~/tutorials/signalr/samples/SignalRChat/Pages/Index.cshtml)]
+[!code-cshtml[Index](~/tutorials/signalr/samples/SignalRChat/Pages/Index.cshtml)]
 
-   The preceding markup:
+The preceding markup:
 
-   * Creates text boxes and a submit button.
-   * Creates a list with `id="messagesList"` for displaying messages that are received from the SignalR hub.
-   * Includes script references to SignalR and the `chat.js` app code is created in the next step.
+* Creates text boxes and a submit button.
+* Creates a list with `id="messagesList"` for displaying messages that are received from the SignalR hub.
+* Includes script references to SignalR and the `chat.js` app code is created in the next step.
 
-1. In the `wwwroot/js` folder, create a `chat.js` file with the following code:
+In the `wwwroot/js` folder, create a `chat.js` file with the following code:
 
-   [!code-javascript[chat](~/tutorials/signalr/samples/SignalRChat/wwwroot/js/chat.js)]
+[!code-javascript[chat](~/tutorials/signalr/samples/SignalRChat/wwwroot/js/chat.js)]
 
-   The preceding JavaScript:
+The preceding JavaScript:
 
-   * Creates and starts a connection.
-   * Adds to the submit button a handler that sends messages to the hub.
-   * Adds to the connection object a handler that receives messages from the hub and adds them to the list.
+* Creates and starts a connection.
+* Adds to the submit button a handler that sends messages to the hub.
+* Adds to the connection object a handler that receives messages from the hub and adds them to the list.
 
 ## Run the app
 
 # [Visual Studio](#tab/visual-studio)
 
-1. Press CTRL+F5 to run the app without debugging.
+Press CTRL+F5 to run the app without debugging.
 
 # [Visual Studio Code](#tab/visual-studio-code)
 
-1. In the integrated terminal, run the following command:
+In the integrated terminal, run the following command:
 
-  ```dotnetcli
-  dotnet watch run --project SignalRChat.csproj
-  ```
+```dotnetcli
+dotnet watch run --project SignalRChat.csproj
+```
 
 # [Visual Studio for Mac](#tab/visual-studio-mac)
 
 1. From the menu, select **Run > Start Without Debugging**.
```

Comment:
Did you leave this number here on purpose?

Reply:
I just missed it somehow, will correct, thanks Tom for spotting it.

Reply:
I guess that was already fixed in commit 5118771 before the review request, but wasn't showing on your end yet.  Already removed.